_data/faqs.yaml

```diff
@@ -23,6 +23,10 @@
   answer: >-
     We have been gathering lots of input on metadata from a diverse set of potential users during the project's Design Phase. We have not yet determined the metadata requirements to be implemented when we first launch Hydra-in-a-Box, though it is likely that we will support the <a href="http://dp.la/about/map/">DPLA MAP v4</a> as default descriptive metadata for easier integration with DPLA hubs. As for crosswalking your metadata (which might be in XML) into a format that Hydra-in-a-Box will use (such as RDF), we're thinking about that, too, and when we have answers, we'll share them. In the meantime, we’d love to hear from you. Please share your suggestions using in our <a href="http://bit.ly/hydrainabox-ideas">Ideas form</a>. You may also <a href="mailto:hybox-contact@googlegroups.com">email us</a> with any questions or to talk to us in more detail about crosswalking/mapping strategies.
 - question: >-
+    Will Hydra-in-a-Box include metadata aggregation tools?
+  answer: >-
+    The short answer is no, however, the Hydra-in-a-Box grant project includes further development of DPLA’s ingestion system, Heiðrún, to create a more robust suite of aggregation tools to be used by DPLA hubs and others engaged in aggregating, remediating, and enriching metadata from multiple sources. While this product will not be tightly coupled with Hydra-in-a-Box, the two will be complementary and may share some componentry. And while it will be easy to use Hydra-in-a-box and the aggregation tools together, the two will be fully featured and operate independently. In other words, Hydra-in-a-Box will not be required to run "Aggregator-in-a-Box" and vice versa. More information about Heiðrún is <a href="https://digitalpubliclibraryofamerica.atlassian.net/wiki/display/TECH/Heidrun">available on the DPLA wiki</a>.
```

Comment:
@hannahfrost @anarchivist Only change I might make is to capitalize "box" in `...easy to use Hydra-in-a-box ...`. Do we care about that or meh?